projects/qemu/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get install -y make autoconf automake libtool libglib2.0-dev libfdt-dev libpixman-1-dev zlib1g-dev patchelf
```

Comment:
Can you use the same packages we use to build mainstream QEMU? That would extend the coverage.
See https://github.com/qemu/qemu/blob/v5.0.0/tests/docker/dockerfiles/debian-amd64.docker:
```
RUN apt update && \
    DEBIAN_FRONTEND=noninteractive eatmydata \
    apt build-dep -yy qemu

RUN apt update && \
    DEBIAN_FRONTEND=noninteractive eatmydata \
    apt install -y --no-install-recommends \
        libbz2-dev \
        liblzo2-dev \
        libgcrypt20-dev \
        librdmacm-dev \
        libsasl2-dev \
        libsnappy-dev \
        libvte-dev
```

Reply:
@a1xndr, feel free to upload a new PR for this packages improvement.